lib/replicache.dart

```diff
@@ -87,6 +87,25 @@ class Replicache implements ReadTransaction {
     await _staticInvoke(name, 'drop');
   }
 
+  /// Sets the verbosity level Replicache logs at. By default,
+  /// Replicache logs at [LogLevel.info].
+  static void set logLevel(LogLevel level) {
```

Comment:
I wasn't able to implement your suggestion because even if we make this a bare function in log.dart, it still needs access to replicache.dart to call `_staticInvoke`.